joebeernink/TrashMob
# Update event details

This PR updates the event details page.

- Created Register button component with add attendee logic.
- Moved attendance list to App so it can be shared. Attendance info should be stored elsewhere in a future iteration (React context).

Test steps:
1. Click Register button for an event on the home page. You should no longer see Register button.
2. Click View button. You should see your username in the attendee list on Event Details page.
3. Click on your username in the nav bar and select My Dashboard. Remove yourself from the event.
4. Return to the event details and click Register button. You should see your username in the attendee list on Event Details page. You should no longer see Register button.
